bin/homeshick

```diff
@@ -23,12 +23,14 @@ exit_status=$EX_SUCCESS
 
 type git &>/dev/null || err "$EX_SOFTWARE" "git not found in path"
 
-mkdir -p "$repos"
+if [[ ! -d $repos ]]; then
+	mkdir -p "$repos"
```

Comment:
Not a Directory? Even if it is possible to have a directory and file inode with the same name in the same location, we should never do it.
So no If-clause need here, mkdir -p already checks if existent.

Reply:
I'm trying to avoid an exec here, but I could change it to `-e` if that helps.

Reply:
If you change it to `-e` you should check if it is a file and bail out with a proper message 
``` bash
if [[ ! -e "$repos" ]]; then
	mkdir -p "$repos"
elif [[ -f "$repos" ]]; then
	err "$EX_ERR" "$repos is a file"
fi
```

Reply:
You are right about the error message but I thought for the sake of performance one wants to check that also with bash builtins.